### PR TITLE
fix: bug when sequence units level appear after tapping back

### PR DESCRIPTION
**Pull request by dixidroid**
Removed sequence units level when open a content from Resume or Dates

### CODE REVIEW

**Review by HamzaIsrar12 — CHANGES_REQUESTED** [review]
We need to make changes while considering the `COURSE_NESTED_LIST_ENABLED` flag, accounting for both true and false values.

Previously, it functioned correctly with the `false` value, but now it operates with the `true` value. Let's account for both scenarios this time. 😄

**Comment by dixidroid** [comment]
@HamzaIsrar12 so, when COURSE_NESTED_LIST_ENABLED = false when an user clicks the resume button the next content will be opened and then when he goes back the list of units should appear? Don't think this is a good behaviour...thoughts?

**Comment by HamzaIsrar12** [comment]
@dixidroid 
It's a reasonable approach. If it seems incomplete, it might be because we previously highlighted the relevant unit on back press, providing a visual cue for the user to identify the unit they were on.

**Comment by dixidroid** [comment]
@HamzaIsrar12 I've added an ability to use the COURSE_NESTED_LIST_ENABLED flag

File: course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineFragment.kt [hunk 1]
```diff
@@ -168,14 +168,24 @@ class CourseOutlineFragment : Fragment() {
                         viewModel.resumeSectionBlock?.let { subSection ->
```
Comment (HamzaIsrar12) [inline course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineFragment.kt:168]: We can also move this and the other block (in DatesFragment) to their respective ViewModel, but that's up to you now. 😄

**Review by HamzaIsrar12 — APPROVED** [review]
LGTM 🏎️ ✨

**Review by volodymyr-chekyrta — APPROVED** [review]

**Review by k1rill — APPROVED** [review]

**Pull request merged**